test/transport.test.js

```diff
@@ -83,6 +102,29 @@ test('pino.transport with package', { skip: isWin }, async ({ same, teardown })
   })
 })
 
+test('pino.transport loads the pino-elasticsearch package', ({ plan, ok, fail, equal, pass }) => {
+  plan(3)
+  const transport = pino.transport({
+    target: 'pino-elasticsearch',
+    options: {
+      node: null // triggers the error if the module is loaded correctly
+    }
+  })
+
+  const instance = pino(transport)
+  instance.info('hello')
+
+  transport.on('ready', function () {
+    fail('ready event should not be emitted')
+  })
+  transport.on('error', function (err) {
+    ok(err)
+    equal(err.message, 'Missing node(s) option', 'pino-elastisearch trigger a valid error')
+  })
+
+  pass('pino instance created')
+})
```

Comment:
I would remove this test. We should ideally cover it with the yarn pnp automatic linking as it's done for transport.

Reply:
Do you mean running `yarn add pino-elasticsearch`?

Right now this test is equal to the `pino-pretty` one, since the last merge on master removed the `#pino-pretty` hard coding case

Reply:
then remove the test completely!